src/MySqlConnector/Core/BinaryRow.cs

```diff
@@ -11,7 +11,7 @@ namespace MySqlConnector.Core;
 internal sealed class BinaryRow : Row
```

Comment:
There's so little code left in these classes they could probably be eliminated.

Will fix in https://github.com/mysql-net/MySqlConnector/pull/1332/commits/acd9b9475bf6841c637b1b7777ddfadd37df7ca5.